### PR TITLE
Orders API

### PR DESCRIPTION
**Pull request by samudary**
Fixes https://github.com/DripEmail/drip/issues/8123

### CODE REVIEW

File: test/drip/resources_test.rb [hunk 1]
```diff
@@ -6,6 +6,7 @@ class Drip::ResourcesTest < Drip::TestCase
     assert_equal Drip::Campaign, Drip::Resources.find_class("campaign")
     assert_equal Drip::CampaignSubscription, Drip::Resources.find_class("campaign_subscription")
     assert_equal Drip::Error, Drip::Resources.find_class("error")
+    assert_equal Drip::Order, Drip::Resources.find_class("order")
     assert_equal Drip::Purchase, Drip::Resources.find_class("purchase")
```
Comment (andrewstucki) [inline test/drip/resources_test.rb]: Here

File: lib/drip/resources.rb [hunk 2]
```diff
@@ -3,6 +3,7 @@
 require "drip/resources/campaign"
 require "drip/resources/campaign_subscription"
 require "drip/resources/error"
+require "drip/resources/order"
 require "drip/resources/purchase"
```
Comment (andrewstucki) [inline lib/drip/resources.rb]: Here

File: lib/drip/resources.rb [hunk 3]
```diff
@@ -19,6 +20,7 @@ def self.classes
         Drip::Campaign,
         Drip::CampaignSubscription,
         Drip::Error,
+        Drip::Order,
         Drip::Purchase,
```
Comment (andrewstucki) [inline lib/drip/resources.rb]: Here

File: lib/drip/client.rb [hunk 4]
```diff
@@ -7,6 +7,7 @@
 require "drip/client/custom_fields"
 require "drip/client/events"
 require "drip/client/forms"
+require "drip/client/orders"
 require "drip/client/purchases"
```
[the remark this answers is not in the document]
Comment (samudary) [reply lib/drip/client.rb]: @andrewstucki Got it, thanks. Completely overlooked that!

**Comment by samudary** [comment]
@andrewstucki Purchases now gone forever

File: lib/drip/client/purchases.rb [hunk 5]
```diff
@@ -1,65 +1,17 @@
...
-      #                            interface to view the order details.
-      #            - occurred_at - Optional. The String time at which the event
-      #                            occurred in ISO-8601 format. Defaults to the
-      #                            current time.
-      #
+      # options  - Required. A Hash of additional order options. Refer to the
+      #                      Drip API docs for the required schema.
       # Returns a Drip::Response.
-      # See https://www.getdrip.com/docs/rest-api#create_purchase
+      # See https://developer.drip.com/#orders
```
Comment (tayjohno) [inline lib/drip/client/purchases.rb]: I'd add a note here about this being deprecated. We should also add in a deprecation warning inside this method directing users to update to purchases instead. https://stackoverflow.com/a/294114

**Comment by samudary** [comment]
@tayjohno Thanks. Completely forgot about that. Updated.

**Pull request merged**